source/pool/contracts/Pool.sol

```diff
@@ -134,6 +137,19 @@ contract Pool is Ownable {
         .div(100);
   }
 
+  function calculateMultiple(uint256 score, address[] calldata tokens)
+    external
+    view
+    returns (uint256[] memory)
+  {
+    uint256[] memory outputAmounts = new uint256[](tokens.length);
+    for (uint256 i = 0; i < tokens.length; i++) {
+      uint256 output = calculate(score, tokens[i]);
+      outputAmounts[i] = output;
+    }
+    return outputAmounts;
```

Comment:
If you name the return value in the signature, you don't need to explicitly return AFAIK. Would there be a benefit?

Reply:
Only style AFAICT. I'll change if you like.